### PR TITLE
Update README for adding testInstrumentationRunnerArguments

### PR DESCRIPTION
**Pull request by Goooler**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,13 +108,15 @@ To start writing instrumentation tests with JUnit Jupiter, make the following ch
...
+  ```kotlin
   android {
     defaultConfig {
       // 1) Make sure to use the AndroidJUnitRunner, or a subclass of it. This requires a dependency on androidx.test:runner, too!
       testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
       // 2) Connect JUnit 5 to the runner
-      testInstrumentationRunnerArgument("runnerBuilder", "de.mannodermaus.junit5.AndroidJUnit5Builder")
+      testInstrumentationRunnerArguments += mapOf(
+        "runnerBuilder" to "de.mannodermaus.junit5.AndroidJUnit5Builder"
+      )
```
Comment (Goooler) [inline README.md:119]:
```kotlin
    @Incubating
    @Deprecated("Replaced by testInstrumentationRunnerArguments property")
    fun testInstrumentationRunnerArguments(args: Map<String, String>)
```

File: README.md [hunk 2]
```diff
@@ -108,13 +108,13 @@ To start writing instrumentation tests with JUnit Jupiter, make the following ch
...
   
-  ```groovy
+  ```kotlin
   android {
     defaultConfig {
       // 1) Make sure to use the AndroidJUnitRunner, or a subclass of it. This requires a dependency on androidx.test:runner, too!
       testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
       // 2) Connect JUnit 5 to the runner
-      testInstrumentationRunnerArgument("runnerBuilder", "de.mannodermaus.junit5.AndroidJUnit5Builder")
+      testInstrumentationRunnerArguments["runnerBuilder"] = "de.mannodermaus.junit5.AndroidJUnit5Builder"
```
Comment (Goooler) [inline README.md:117]:
```kotlin
    @Incubating
    @Deprecated("Replaced by testInstrumentationRunnerArguments property")
    fun testInstrumentationRunnerArguments(args: Map<String, String>)
```

**Comment by mannodermaus** [comment]
Thanks for this. Has this deprecation been introduced in AGP 7? The JUnit 5 plugin supports all the way down to AGP 4.0 and I'd like the readme to reflect the lowest common denominator in terms of how it's written. Furthermore, the readme is generated from a template file - please apply modifications  to `README.md.template` as well.

**Comment by Goooler** [comment]
Yep, it's from AGP 7.0.

**Comment by mannodermaus** [comment]
Looks good, and thanks for updating the language of that code block, too!

**Pull request merged**